Lib/glyphsLib/builder/features.py

```diff
@@ -140,16 +140,19 @@ def _to_ufo_features(
                     feature_names = ["featureNames {", f'  name "{name}";', "};"]
         elif font.format_version == 3 and feature.labels:
             feature_names = []
-            feature_names.append("featureNames {")
             for label in feature.labels:
                 langID = _to_name_langID(label["language"])
                 name = label["value"]
                 name = name.replace("\\", r"\005c").replace('"', r"\0022")
+                if name == "":
```

Comment:
How about you move this one line up? The replacements don't affect the emptiness of the string and can be skipped

Reply:
🤣 your wish is my command